integration_test.go

```diff
@@ -194,6 +194,57 @@ func TestIntegrationFullOperatorSetWithRetry(t *testing.T) {
 
 }
 
+func TestUpdateStakes(t *testing.T) {
+	/* Start the anvil chain */
+	anvilC := startAnvilTestContainer()
+	// Not sure why but deferring anvilC.Terminate() causes a panic when the test finishes...
+	// so letting it terminate silently for now
+	anvilHttpEndpoint, err := anvilC.Endpoint(context.Background(), "http")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	contractAddresses := getContractAddressesFromContractRegistry(anvilHttpEndpoint)
+	operatorEcdsaPrivKeyHex := "ac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"
+	operatorEcdsaPrivKey, err := crypto.HexToECDSA(operatorEcdsaPrivKeyHex)
+	if err != nil {
+		t.Fatal(err)
+	}
+	operatorAddr := crypto.PubkeyToAddress(operatorEcdsaPrivKey.PublicKey)
+	operatorBlsPrivKey := "0x1"
+	avsSync := NewTestAvsSync(anvilHttpEndpoint, contractAddresses, []common.Address{})
+
+	// first register operator into avs. at this point, the operator will have whatever stake it had registered in eigenlayer in the avs
+	registerOperatorWithAvs(anvilHttpEndpoint, contractAddresses, operatorEcdsaPrivKeyHex, operatorBlsPrivKey)
+
+	// get stake of operator before sync
+	operatorsPerQuorumBeforeSync, err := avsSync.avsReader.GetOperatorsStakeInQuorumsAtCurrentBlock(&bind.CallOpts{}, []byte{0})
+	if err != nil {
+		t.Fatal(err)
+	}
+	// TODO: should be checking all operators, not just the first one
+	operatorStakeBeforeSync := operatorsPerQuorumBeforeSync[0][0].Stake
+
+	// deposit into strategy to create a diff between eigenlayer and avs stakes
+	depositAmount := big.NewInt(100)
+	depositErc20IntoStrategyForOperator(anvilHttpEndpoint, contractAddresses.DelegationManager, contractAddresses.Erc20MockStrategy, operatorEcdsaPrivKeyHex, operatorAddr.Hex(), depositAmount)
+
+	avsSync.updateStakes()
```

Comment:
would prefer to test setting SyncInterval to 0 instead of doing this like this. Better in general I feel to only test the public functions only (aka `Start()` and not the internal ones). This way we keep it to a single entry point.